src/core/types.h

```diff
@@ -434,6 +434,7 @@ class Box {
     // Add a no-op constructor to make sure that we don't zero-initialize cls
     Box() {}
 
+    void* operator new(size_t size, BoxedClass* cls, size_t nitems) __attribute__((visibility("default")));
```

Comment:
oh I just realized you were commenting on a line I had added in this diff. >_< I'm so silly